crates/sncast/src/starknet_commands/account/add.rs

```diff
@@ -52,16 +51,16 @@ pub struct Add {
     #[clap(short, long)]
     pub salt: Option<FieldElement>,
 
-    /// If passed, a profile with corresponding data will be created in Scarb.toml
+    /// If passed, a profile with corresponding data will be created in snfoundry.toml
```

Comment:
nit:
```suggestion
    /// If passed, a profile with the provided name and corresponding data will be created in snfoundry.toml
```

Reply:
changed (also in account/create) https://github.com/foundry-rs/starknet-foundry/pull/1644/commits/321691a0c92cb4c5890734e9a931c1fd2c806607